### DISCUSSION TITLE
Can we sponsor in exchange for a feature being implemented?

### DISCUSSION THREAD

**Discussion by liamsharp** [pmd/pmd#4179] [General]
I'd love for https://github.com/pmd/pmd/issues/2867 to be implemented and would be willing to pay rather than do it myself - is this possible?

**Comment by adangel** [comment]
Hi @liamsharp , thanks for your interest.
You can of course sponsor PMD, either on GitHub (https://github.com/sponsors/pmd) or on OpenCollective (https://opencollective.com/pmd).
However, the reality is, that PMD doesn't have any full-time developers working paid full-time on the project. It's all done in free time.

I can put the issue on my TODO list, but I won't promise anything.

**Reply by adangel** [reply]
see #4195

**Reply by liamsharp** [reply]
Fantastic, thank you! We'll test this out soon :-) We've also followed up by sponsoring.

**Reply by adangel** [reply]
Thanks for sponsoring PMD! :heart:
